airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -168,8 +168,11 @@ protected String primaryKeyClause(final List<String> columns) {
     return clause.toString();
   }
 
+  protected void customSetup() {}
```

Comment:
looks like this is only used for Teradata, an only to create a database object. Any reason we can't do the same thing in the TeradataTestContainer constructor (which is only used by the TeradataAcceptanceTest) ?

Reply:
Its being used by Teradata and Redshift. Most of these changes that you see is a band-aid trying to make the JdbcSourceAcceptanceTest work for these un-certified connectors.